### ISSUE TITLE
Error When Attempting a Deployment

### ISSUE THREAD

**Issue by leogopal** [bradp/vv#60]
I run ```vv deployment-create```

I follow all the prompts. Run the command to deploy, then the following error graces my terminal.

```
Leos-MacBook-Pro:vagrant-local leogopal$ vv -v push gfntv-staging
There is a syntax error in the following Vagrantfile. The syntax error
message is reproduced below for convenience:

/Users/leogopal/vagrant-local/Vagrantfile:221: syntax error, unexpected keyword_end, expecting end-of-input
;
```

**Comment by clreed87** [comment]
I'm getting the same error trying to set up my deployments.  Any idea what I'm doing wrong in my configuration?

**Comment by bradp** [comment]
What version are you running? @leogopal @clreed87 
 
This was an issue that was fixed in the most recent versions.

**Comment by leogopal** [comment]
```1.5.0```

**Comment by clreed87** [comment]
Same here: 1.5.0

**Comment by bradp** [comment]
Can you send me your Vagrantfile? (with any FTP passwords or hosts edited out)

Thanks.

**Comment by leogopal** [comment]
Here you go: [Vagrantfile](https://www.dropbox.com/s/8pjvd5x5awv18ll/Vagrantfile?dl=)

**Comment by bradp** [comment]
@leogopal I dont see an Line 221 in your Vagrantfile, or any deployment setup.

**Comment by clreed87** [comment]
Sure you can take a look at mine: (Vagrantfile)[https://www.dropbox.com/s/67kudqvxyahkkeq/Vagrantfile?dl=0]

**Comment by bradp** [comment]
Looks like something is getting caught up.

I'm not seeing ``"# end-vv-$site-$deployment_name"`` at the end of the deployment, which should be printed out. It may be conflicting with vagrant-triggers, which I haven't tested much with.

I'll do some testing and trying to figure it out. Thanks for your help.

**Comment by clreed87** [comment]
No problem. Thats for taking a look. Let me know if there is any additional testing I can do or anything else I can do to help.  

My VVV setup is pretty much straight out of the box, so nothing should be too weird.  Just installed VVV and Variable VV, used <code>vv create</code> to create the sites within my <code>www</code> directory, and then used <code>vv deployment-create</code> to test out the deployment feature.

**Comment by evelynriossf** [comment]
I'm getting a similar error:

There is a syntax error in the following Vagrantfile. The syntax error message is reproduced below for convenience:

/Users/evelynrios/Developer/vagrant-local/Vagrantfile:248: syntax error, unexpected keyword_end, expecting end-of-input

I dug into the Vagrant file, and it looks like VV is putting the push config "after" the end of the main script. I deleted the extra end command and the error went away.

**Comment by bradp** [comment]
Yeah, it should be replacing the last "end" in the config, rather than appending to it.

**Comment by bradp** [comment]
Fixed in e570e12b71347118e849a6d4996a09fc6702728f. Will be in the next version deployed later this week.
